core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -164,16 +183,17 @@ oppia.controller('FeedbackTab', [
             var UNSAVED_CHANGES_MSG = 'You have unsaved changes to ' +
               'this exploration. Please save/discard your unsaved changes if ' +
               'you wish to accept.';
-            $scope.isOpen = suggestionIsOpen;
+            $scope.isNotHandled = !suggestionIshandled;
```

Comment:
Typo: Should be `suggestionIsHandled` (capital H) right?

Reply:
Done thanks!